doc/developer.md

```diff
@@ -64,12 +64,18 @@ tox -e dev
 . .tox/dev/bin/activate
 ```
 
-Finally, ou can also explicitly install all the Python dependencies for sourmash by running
+Finally, you can also explicitly install all the Python dependencies for sourmash by running
 ```
 pip install -r requirements.txt
 ```
 (but they are already installed in the virtualenv created with `tox -e dev`).
 
+## Updating your developer environment
+
+To update rust to the latest version, use [`rustup update`].
```

Comment:
curious about the use of square brackets here - is there a reason to go with this convention? I don't think we use it anywhere else in the code base.

Reply:
I think I must have put it there by mistake. I'll remove it right away.